src/main/java/nl/cherement/jak/controller/UserController.java

```diff
@@ -1,52 +1,58 @@
 package nl.cherement.jak.controller;
 
 import nl.cherement.jak.entity.UserEntity;
-import nl.cherement.jak.service.UserPrincipalDetailsService;
+import nl.cherement.jak.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.web.bind.annotation.*;
 
 import javax.servlet.http.HttpServletRequest;
 import java.util.List;
 import java.util.Optional;
 
-@CrossOrigin(origins = "*", allowedHeaders = "*")
+
 @RestController
 @RequestMapping("/user")
 public class UserController {
 
 
     @Autowired
-    private UserPrincipalDetailsService userPrincipalDetailsService;
+    private UserService userService;
+
+
+
 
 
     @GetMapping
     public List<UserEntity> findAll(HttpServletRequest request) {
-        return userPrincipalDetailsService.findAll();
+        return userService.findAll();
```

Comment:
de findAll en find kunnen eigenlijk wel weg nu. Je hoort toch geen users op te kunnen halen

Reply:
Mee eens ze zijn er uitgehaald. Users worden dan alleen nog meegeven als een board word opgevraagd. De `getUser()` is nog wel nodig om de User zichzelf op te kunnen vragen